x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/page.tsx

```diff
@@ -29,9 +29,13 @@ import { DataFrameAnalyticsList } from './components/analytics_list';
 import { RefreshAnalyticsListButton } from './components/refresh_analytics_list_button';
 import { useRefreshInterval } from './components/analytics_list/use_refresh_interval';
 import { useCreateAnalyticsForm } from './hooks/use_create_analytics_form';
+import { AnalyticStatsBarStats, StatsBar } from '../../../components/stats_bar';
 
 export const Page: FC = () => {
   const [blockRefresh, setBlockRefresh] = useState(false);
+  const [stats, setStats] = useState<AnalyticStatsBarStats | undefined>(
+    {} as AnalyticStatsBarStats
```

Comment:
i don't follow why `stats` needs to be `AnalyticStatsBarStats | undefined`.
when will it be `undefined`?